plugins/clickhouse-monitor/main.go

```diff
@@ -60,6 +64,21 @@ var (
 	monitorExecInterval time.Duration
 )
 
+var notAValidIdentifierError = errors.New("not a valid identifier")
+
+func sanitizeIdentifier(identifier string) (string, error) {
```

Comment:
Clickhouse identifiers cannot be equal to keywords either, maybe worth adding a comment that we only check the regex here.
https://clickhouse.com/docs/en/sql-reference/syntax/#identifiers

Reply:
sure, I'll add a comment. it's unpractical to check for keywords here, and there is no risk of SQl injection with a single keyword (no space), it will be rejected by ClickHouse

Reply:
done